source/_templates/layout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "!layout.html" %}
+{% block content %}
+{% if current_version and latest_version and current_version != latest_version %}
+<div class="rst-content">
+  <div class="admonition warning">
+    <p class="admonition-title">Warning</p>
+    <p>
+      {% if current_version.is_released %}
+      You're reading the manual for Mixxx {{current_version.version}}.
+      If you're using another version, please select to another version in the sidebar.
```

Comment:
please select to another -> please select another

But maybe the overall flow would be smoother as:

"If you're using a different version you can switch to it in the sidebar."